src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets

```diff
@@ -62,7 +62,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(UseCurrentRuntimeIdentifier)' == 'true'">
-    <RuntimeIdentifier>$(NETCoreSdkRuntimeIdentifier)</RuntimeIdentifier>
+    <RuntimeIdentifier>$(NETCoreSdkPortableRuntimeIdentifier)</RuntimeIdentifier>
```

Comment:
On Alpine Linux,  this broke a project I was porting to .NET 7:

```sh
$ dotnet publish -o dist --use-current-runtime -p:PublishSingleFile=true --self-contained -v:diag | grep -E '(NETCoreSdkPortableRuntimeIdentifier|NETCoreSdkRuntimeIdentifier) =' | sort -u
                   NETCoreSdkPortableRuntimeIdentifier = linux-arm64
                   NETCoreSdkRuntimeIdentifier = linux-musl-arm64
```

Reply:
`NETCoreSdkPortableRuntimeIdentifier` should be `linux-musl-arm64` on `Alpine`.